docs/application/dotnet/guides/nui/xaml/layout-defining-example.md

```diff
@@ -0,0 +1,149 @@
+# Layout Defining Example
```

Comment:
Suggestion:

Layout Define Example

OR

Define Layout Example

Please verify and suggest.